pages/blog/blog-post-template.html

```diff
@@ -24,10 +24,16 @@ <h2>Blog Post Title Written by the Author</h2>
                 <p>Posted by AUTHOR NAME on POSTED DATE</p>
             </div>
         </section>
-        <img src="https://www.pngkey.com/png/detail/233-2332677_image-500580-placeholder-transparent.png" alt="A generic body image and alt text">
-        <h3>Placeholder Section Title</h3>
-        <p>This is a placeholder paragraph. You should replace it with your own blog post text. If you need more paragraphs, add more p tags below this one. If you need more sections, add more h3 tags below this one.</p>
-        <footer-component></footer-component>
+        
+        <article class="blog_post_article">
+            <img src="https://www.pngkey.com/png/detail/233-2332677_image-500580-placeholder-transparent.png" alt="A generic body image and alt text">
+            <h3>Placeholder Section Title</h3>
+            <p>
+                This is a placeholder paragraph. You should replace it with your own blog post text. 
+                If you need more paragraphs, add more p tags below this one. 
+                If you need more sections, add more h3 tags below this one.
+            </p>
+        </article>
```

Comment:
I thought it'd be nice if we wrapped our blog post content inside of an `<article>` element 😄 I think this one might be new to you? If it is, here is a little read about them: [https://www.w3schools.com/tags/tag_article.asp](https://www.w3schools.com/tags/tag_article.asp)

Inside of `<article>` we're still allowed to use any `<div>`, `<section>` or anything else to help us wrap up our content into sections, columns, and similar 💯